src/main/java/base/HelloWorld.java

```diff
@@ -0,0 +1,43 @@
+package base;
+
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String str;
```

Comment:
What contains variable with name `str`? It is some message or keys for launching rockets?